app/views/admin/events/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>
+  Event list
+  <a href="events/new" class="btn btn-primary active" role="button" style="float: right;">New</a>
+</h1>
+
+
+
+
+<table class="table table-striped table-hover">
+  <tr>
+    <th>Meetup</th>
+    <th>URL</th>
+    <th>Actions</th>
+  </tr>
+
+  <% @events.each do |event| %>
+      <tr>
+        <td><%= event.title %></td>
+        <td>
+          <%= link_to event.meetup_url, event.meetup_url, target: '_blank' %>
+        </td>
+        <td>
+          <%= link_to "events/edit/#{event.id}" do %>
```

Comment:
Here to.. Use the `_path` helper instead using a fixed String as path.
Check the path with `docker-compose run web bin/rake routes`